src/profiles/profileSchemas/person.ts

```diff
@@ -111,7 +110,7 @@ export class Person extends Profile {
 
   static validateSchema(profile: any, strict = false) {
     schemaDefinition.strict = strict
-    return inspector.validate(schemaDefinition, profile)
+    return new Ajv().validate(schemaDefinition, profile)
```

Comment:
A few of the schema validation tests are failing -- the tape unit test error messages are not that helpful by default:
```
not ok 120 Person profile should be valid
  ---
    operator: ok
    expected: true
    actual:   undefined

not ok 120 Person profile should be valid
  ---
    operator: ok
    expected: true
    actual:   undefined

not ok 138 Profile should be in a valid format
  ---
    operator: ok
    expected: true
    actual:   false

```

Looks like the `avj` config and return properties are different. For example this `strict` option appears to be specified in a different way in avj.

And the validation return object has different properties. See this code from the unit tests: 

```
    const validationResults = Profile.validateSchema(sampleProfiles.naval)
    t.ok(validationResults.valid, 'Profile should be valid')
```
I don't think `result.valid` is a defined property. And looks like there are some options for reporting better error messages (from avj readme):
```js
var valid = ajv.validate(schema, data);
if (!valid) console.log(ajv.errors);
```